util/secscan/v4/api.py

```diff
@@ -197,6 +218,20 @@ def _perform(self, action):
 
         return resp
 
+    def _sign_jwt(self):
+        """
+        Sign and return a jwt.
+
+        If self.jwt_psk is provided a pre-shared key will be used as the signing key.
+        """
+        payload = {
+            "iss": "quay",
+            "exp": datetime.utcnow() + timedelta(minutes=5),
```

Comment:
We should make this timedelta threshold configurable. Not necessary for this PR but probably good for inclusion as a TODO comment.